### ISSUE TITLE
Input lag

### ISSUE THREAD

**Issue by soshimee** [ChristopherHX/mcpelauncher-manifest#262]
This worked fantastic on my laptop but it doesn't work so well on my desktop PC, it has a lot of input delay which is easily noticeable and I can't pvp on bedrock anymore.

**Comment by soshimee** [comment]
It seems that the fps is being limited to 60.

**Comment by soshimee** [comment]
Fiddling around with options doesn't help either.

**Comment by ChristopherHX** [comment]
> it has a lot of input delay which is easily noticeable 

Are you shure you are running the game without software rendering on a real gpu?
Otherwise your PC is weaker than your laptop.

> It seems that the fps is being limited to 60.

This is vsync, the game runs always @ 60 fps if your hardware is fast enough.
Turning of vsync will allow 100+ fps (if the fps limit isn't enabled) these values are both in the minecraft's option.txt and inaccessible from game settings.

**Comment by soshimee** [comment]
Yes, I already fiddled around with option.txt but it doesn't do anything. The exact option was the vsync one which was set to 1 and I changed it to 0, also I changed the max framerate option too but reverted it back since it had no effect like the vsync one.

**Comment by ChristopherHX** [comment]
I have no idea what you did, but it work for me.
![vsyncoff](https://user-images.githubusercontent.com/44845461/120473940-da0a6080-c3a7-11eb-93ba-f08b5e9ac624.png)
This won't increase fps to 700+, if your system cannot even reach 60 fps.
I doub't this launcher has a significant input problem.
You see the fps at the  top of Minecraft startscreen, because this is a beta of 1.17 and it worked for older versions too.

**Comment by soshimee** [comment]
My PC is much more powerful than my laptop, Minecraft Java runs much faster on my PC than on my laptop.

**Comment by soshimee** [comment]
Also I can't even use versions 1.15+, has there been an update?

**Comment by soshimee** [comment]
I can limit my FPS to 60- but not 60+.

**Comment by soshimee** [comment]
Is there another options file (the one I edited could have been a different one), if so where is it?

**Comment by soshimee** [comment]
![image](https://user-images.githubusercontent.com/45235204/120588457-2e7a1280-c472-11eb-8951-aea40d7ee8f9.png)
gfx_max_framerate set to 60- works but 60+ gets ignored and vsync is used

**Comment by ChristopherHX** [comment]
You are really using an ancient version of this launcher.
- If the version is from 2019 then 1.14.1.5 is the max version
- If the version is from 2019 and 1.14.1.4 work then it is outdated too, but my fork
- If the version is from 2020-6 then 1.16.0 works
- If the version is from 2021-6 then 1.17.0 works

**Comment by soshimee** [comment]
I tried getting the latest version, but it seems to be same.

**Comment by soshimee** [comment]
How do I enable Beta? Now that I have updated, Beta is disabled and I cannot see the fps.

**Comment by soshimee** [comment]
I think I solved it. There was another config file, however, it could be just my feelings because I cannot use beta.

**Comment by GameParrot** [comment]
This might be related to [MCPE-98861](https://bugs.mojang.com/browse/MCPE-98861)

**Comment by soshimee** [comment]
Different since it also affects lower versions
